static/app/components/hotkeys.tsx

```diff
@@ -0,0 +1,106 @@
+import styled from '@emotion/styled';
+
+import space from 'sentry/styles/space';
+import {getKeyCode} from 'sentry/utils/getKeyCode';
+
+const macModifiers = {
+  18: '⌥',
+  17: '⌃',
+  91: '⌘',
+};
+
+const normalModifiers = {
+  18: 'ALT',
+  17: 'CTRL',
+  91: 'CMD',
+};
+
+const genericGlyphs = {
+  16: '⇧',
+  8: '⌫',
+  37: '←',
+  38: '↑',
+  39: '→',
+  40: '↓',
+  107: '+',
+};
+
+const keyToDisplay = (
+  key: string,
+  isMac: boolean
+): {label: React.ReactNode; specificToOs: 'macos' | 'generic'} => {
+  // Handle escaped + case
+  key = key === '\\+' ? '+' : key;
+
+  const keyCode = getKeyCode(key);
+
+  // Not a special key
+  if (!keyCode) {
+    return {label: <Key>{key.toUpperCase()}</Key>, specificToOs: 'generic'};
+  }
+
+  const modifierMap = isMac ? macModifiers : normalModifiers;
+  const keyStr = modifierMap[keyCode] ?? genericGlyphs[keyCode] ?? key.toUpperCase();
+
+  const specificToOs = keyStr === 'CMD' ? 'macos' : 'generic';
```

Comment:
oh wait. Does this actually work? won't the macos keystr be `⌘`

Reply:
Oh yeah, let me fix that, this was from the remnants of when it would render `CMD` on windows